src/main/java/frc/trigon/robot/subsystems/pitcher/PitcherConstants.java

```diff
@@ -0,0 +1,132 @@
+package frc.trigon.robot.subsystems.pitcher;
+
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.hardware.phoenix6.cancoder.CANcoderEncoder;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXMotor;
+import frc.trigon.robot.hardware.phoenix6.talonfx.TalonFXSignal;
+import frc.trigon.robot.hardware.simulation.SingleJointedArmSimulation;
+import frc.trigon.robot.utilities.mechanisms.SingleJointedArmMechanism2d;
+
+public class PitcherConstants {
+    private static final int
+            MOTOR_ID = 0,
+            ENCODER_ID = 0;
+    private static final String
+            MOTOR_NAME = "PitcherMotor",
+            ENCODER_NAME = "PitcherEncoder";
+    static final TalonFXMotor MOTOR = new TalonFXMotor(MOTOR_ID, MOTOR_NAME, RobotConstants.CANIVORE_NAME);
+    static final CANcoderEncoder ENCODER = new CANcoderEncoder(ENCODER_ID, ENCODER_NAME, RobotConstants.CANIVORE_NAME);
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.Clockwise_Positive;
```

Comment:
Again, I thought we said we're having a blank line between motor declaration and motor settings.